tsconfig.json

```diff
@@ -0,0 +1,36 @@
+{
+  "compilerOptions": {
+    "module": "commonjs",
+    "target": "es2019",
+    "strict": true,
+    "alwaysStrict": false,
+    "strictPropertyInitialization": true,
```

Comment:
a bunch of these options fall under the `strict` flag, so you dont have to explicitly enable them. but there is no harm in doing so